base/docs/basedocs.jl

```diff
@@ -1261,4 +1261,126 @@ available in the `.error` field.
 """
 InitError
 
+"""
+    Any::DataType
+
+`Any` is the union of all types. It has the defining property `isa(x, Any) == true` for any `x`. `Any` describes
+therefore the entire universe of possible values. For example `Integer` is a subset of `Any` that includes `Int`,
+`Int8`, and other concrete types.
+"""
+Any
+
+"""
+    Union{}
```

Comment:
Perhaps collapse this with the docstring of `Union{Types...}` and mention there that `Union{}` is the empty case?